### DISCUSSION TITLE
How to get the Galaxy Brain achievement badge?

### DISCUSSION THREAD

**Discussion by alexanderzzzz** [community/community#20950] [Other Feature Feedback, Questions, & Ideas] [answered]
How to get the Galaxy Brain achievement badge?

**Comment by aaronlamz** [comment, marked as answer]
Hi, to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

* Discussions should be in a public repository
* Get marked as answers to 2 or more questions
* The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
* There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
* Reference: #18454
Hope these steps help you in achieving your badge! Good luck :)

More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by alexanderzzzz** [reply]
thanks!

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
